src/css/mapbox-gl.css

```diff
@@ -60,6 +60,19 @@
 .mapboxgl-ctrl-bottom-left  { bottom: 0; left: 0; }
 .mapboxgl-ctrl-bottom-right { right: 0; bottom: 0; }
 
+.mapboxgl-ctrl-full-container {
+    position: absolute;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
```

Comment:
is `height: 100%` 100% of the height or is it 100% of the width? I thought it meant 100% of the width (even though it's the height property) and that you have to do `bottom: 0` to get full height. Though maybe because it's absolute positioned, this does work.

Reply:
Ah I did not know this, I originally had top: 0; left: 0; bottom: 0; right: 0; but based on a preliminary search it seems documented that height: 100% is 100% of the parent element height?